src/test/java/com/virtuslab/archunit/ClassStructureTestSuite.java

```diff
@@ -45,6 +45,7 @@ public void check(JavaClass javaClass, ConditionEvents events) {
         })
         .because("the SLF4J logger name should reflect the name of the concrete class, not an abstract base")
         .check(importedClasses);
+    throw new Exception();
```

Comment:
Yeah... the tests in top-level project are still left to be migrated.

I'm curious though, how JUnit 4 still made its way to the classpath 🤔

Reply:
Awww crap

<img width="435" alt="image" src="https://user-images.githubusercontent.com/3383210/214982847-14f97d1c-7288-4097-a721-6df3067ae40e.png">